scripts/generate-demo-specs.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
general comment. we should prefix "demo" filenames and targets with the name of the demo (e.g., openapi).

Reply:
done